plugins/auth/src/pam_password.cpp

```diff
@@ -113,28 +116,31 @@ namespace
         //
         // The list of arguments must be terminated by a NULL pointer, and, since these are
         // variadic functions, this pointer must be cast (char *) NULL.
-        execl(PAM_AUTH_CHECK_PROG, PAM_AUTH_CHECK_PROG, _username.c_str(), (char*)NULL);
+        execl(PAM_AUTH_CHECK_PROG, PAM_AUTH_CHECK_PROG, _username.c_str(), (char*) NULL);
 
         THROW(SYS_FORK_ERROR, fmt::format("execl failed [errno:{}]", errno));
-    } // run_pam_auth_check
+    }  // run_pam_auth_check
 #endif // #ifdef RODS_SERVER
 } // anonymous namespace
 
 namespace irods
 {
-    class pam_authentication : public irods_auth::authentication_base {
+    class pam_password_authentication : public irods_auth::authentication_base
+    {
     private:
         static constexpr char* perform_native_auth = "perform_native_auth";
 
     public:
-        pam_authentication()
+        // clang-format off
```

Comment:
I'm guessing these are for the calls to `add_operation()`. If so, let's move them directly around those calls instead of having them around the constructor.

If more code is ever added inside of the constructor, we don't want it to be ignored.

Reply:
Yep, done.